src/main/java/io/dropwizard/metrics/NetuitiveMetricsReporter.java

```diff
@@ -139,14 +140,143 @@ public NetuitiveMetricsReporter build(final String ananke) {
      */
-    @Override
+    @SuppressWarnings("rawtypes")
+	@Override
     public void report(SortedMap<String, Gauge> gauges,
                        SortedMap<String, Counter> counters,
                        SortedMap<String, Histogram> histograms,
                        SortedMap<String, Meter> meters,
                        SortedMap<String, Timer> timers) {
+    	
+    	
+    	LOG.info("Pushing {} gauges, {} counters, {} histograms, {} meters and {} timers to Netuitive",
+    			(gauges == null ? 0 : gauges.size()), 
+    			(counters == null ? 0 : counters.size()), 
+    			(histograms == null ? 0 : histograms.size()), 
+    			(meters == null ? 0 : meters.size()), 
+    			(timers == null ? 0 : timers.size()) 
+    			);
+
+        if (!gauges.isEmpty()) {
+        	LOG.debug("Gauges");
+            for (Map.Entry<String, Gauge> entry : gauges.entrySet()) {
+            	String metricName = entry.getKey();
+            	if (NumberUtils.isNumber(entry.getValue().getValue().toString()))
+            	{
+	            	double metricValue = Double.parseDouble(entry.getValue().getValue().toString());
+	            	pushGauge(metricName, metricValue);
+            	}
+
+            }
+        }
+
+        if (!counters.isEmpty()) {
+        	LOG.debug("Counters");
+            for (Map.Entry<String, Counter> entry : counters.entrySet()) {
+            	String metricName = entry.getKey();
+            	long metricValue = entry.getValue().getCount();
+            	pushCounter(metricName, metricValue);
+            }
+        }
+
+        if (!meters.isEmpty()) {
+        	LOG.debug("Meters");
+            for (Map.Entry<String, Meter> entry : meters.entrySet()) {
+            	String metricName = entry.getKey();
+            	pushMeter(metricName, entry.getValue());
+            }
+        }
+
+        if (!timers.isEmpty()) {
+        	LOG.debug("Timers");
+            for (Map.Entry<String, Timer> entry : timers.entrySet()) {
+            	String metricName = entry.getKey();
+            	pushMeter(metricName, entry.getValue());
+            	pushSnapshot(metricName, entry.getValue().getSnapshot());
+            }
+        }
 
+        if (!histograms.isEmpty()) {
+        	LOG.debug("Histograms");
+            for (Map.Entry<String, Histogram> entry : histograms.entrySet()) {
+            	String metricName = entry.getKey();
+            	pushSnapshot(metricName, entry.getValue().getSnapshot());
+            }
+        }
 
+    }
+    
+	/**
+	 * Pushes the Metered metrics as Gauges
+	 * 
+	 * @param metricName
+	 *            Base metric name for this Meter
+	 * @param meter
+	 *            Metered object to report
+	 */
+    void pushMeter(String metricName, Metered meter)
+    {
+    	pushGauge(metricName + ".count", new Double(meter.getCount()));
+    	pushGauge(metricName + ".meanRate", meter.getMeanRate());
+    	pushGauge(metricName + ".1MinuteRate", meter.getOneMinuteRate());
```

Comment:
Should metric names have any caps in them? I tend to see only lower case metric FQNs.